cribl_stream/README.md

```diff
@@ -0,0 +1,70 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
+
+- Add context to your data, by enriching it with information from external data sources.
+- Help secure your data, by redacting, obfuscating, or encrypting sensitive fields.
+- Optimize your data, per your performance and cost requirements.
+
+Visualize your Cribl Stream metrics in Datadog with the out-of-the-box dashboard.
```

Comment:
```suggestion
Use the out-of-the-box dashboard to view the performance of the Stream with base metrics like events per second, bytes per second, input types, output types, and infrastructure metrics. Monitor reduction percentages by events or bytes, which is useful for improving search performance or licensing and infrastructure costs for the systems of analysis.

```
Let's add this description to the Overview tab - it will not render properly under the Data Collected tab